scripts/release/utils/https_helper.py

```diff
@@ -1,7 +1,7 @@
 import httplib
 
 
-def request(method, host, path, body = None, headers = {}):
+def request(method, host, path, body=None, headers={}):
```

Comment:
It's also in the Python's style guide. I wasn't using IDEA last time, so I missed this.